app/gateway-oss/2.3.x/upgrading.md

```diff
@@ -137,56 +135,56 @@ the database as it is migrated while the old Kong cluster keeps working until
 it is time to decommission it. For this reason, the migration is split into
 two steps, performed via commands `kong migrations up` (which does
 only non-destructive operations) and `kong migrations finish` (which puts the
-database in the final expected state for Kong 2.2.0).
+database in the final expected state for Kong 2.3.0).
 
-1. Download 2.2.0, and configure it to point to the same datastore
+1. Download 2.3.0, and configure it to point to the same datastore
    as your old (1.0 to 2.0) cluster. Run `kong migrations up`.
-2. Once that finishes running, both the old (pre-2.1) and new (2.2.0)
-   clusters can now run simultaneously. Start provisioning 2.2.0 nodes,
+2. Once that finishes running, both the old (pre-2.1) and new (2.3.0)
+   clusters can now run simultaneously. Start provisioning 2.3.0 nodes,
    but do not use their Admin API yet. If you need to perform Admin API
    requests, these should be made to the old cluster's nodes. The reason
    is to prevent the new cluster from generating data that is not understood
    by the old cluster.
 3. Gradually divert traffic away from your old nodes, and into
-   your 2.2.0 cluster. Monitor your traffic to make sure everything
+   your 2.3.0 cluster. Monitor your traffic to make sure everything
    is going smoothly.
-4. When your traffic is fully migrated to the 2.2.0 cluster,
+4. When your traffic is fully migrated to the 2.3.0 cluster,
    decommission your old nodes.
-5. From your 2.2.0 cluster, run: `kong migrations finish`.
+5. From your 2.3.0 cluster, run: `kong migrations finish`.
    From this point on, it will not be possible to start
    nodes in the old cluster pointing to the same datastore anymore. Only run
    this command when you are confident that your migration
    was successful. From now on, you can safely make Admin API
-   requests to your 2.2.0 nodes.
+   requests to your 2.3.0 nodes.
 
 **Cassandra**
 
-Due to internal changes, the table schemas used by Kong 2.2.0 on Cassandra
+Due to internal changes, the table schemas used by Kong 2.3.0 on Cassandra
```

Comment:
I'm not sure that the Cassandra incompatibility is an issue for 2.3. It was at a certain point in time. We will need engineering to verify. @RobSerafini? @tyler-ball ?

Reply:
The aforementioned incompatibility happens when updating from 2.1.0 (or lower) to either 2.2 or 2.3